src/app/tests/suites/certification/Test_TC_DGGEN_2_1.yaml

```diff
@@ -436,7 +457,7 @@ tests:
               - name: "expectedValue"
                 value: "y"
 
-    - label: "Wait for the commissioned device to be retrieved"
+    - label: "Step 10b: Recommission the DUT after a factory reset"
```

Comment:
See above: this needs actual commissioning.

Reply:
Reverted back to the previous version.